OWNERS

```diff
@@ -0,0 +1,9 @@
+approvers:
```

Comment:
Is there a reasons for the `OWNERS` file? This is used for k8s merge automation which is not turned on here.

Reply:
Removed it.